docs/fundamentals/systems/form/styling.md

```diff
@@ -37,52 +37,56 @@ export const label = () => html`
 `;
 ```
 
-## Prefix
+## Input additions
+
+When using an addition to the input make sure to add `data-label` to add the content connected to the input via `aria-labelledby` or add `data-description` to connect it via the `aria-describedby` when the content is relevant for the screen-reader user.
+
```

Comment:
It might seem a bit now if [data-label] is always needed, but that's probably not the case

Reply:
I mean a suffix can also be purely visual (in case of a search icon for a field already labelled "Search") and we don't need data-label or data-description

Reply:
I changed the text to `you could` instead of `make sure to`, but i could make it more explicit.